governance/polls/Proposal - Adjust the BAT Risk Premium

```diff
@@ -0,0 +1,37 @@
+---
+title: Adjust the BAT Risk Premium - August 10, 2020
```

Comment:
Can you use BAT-A rather than BAT, please? I know it's obvious in this case, but we should get into the habit of using the  full vault type identifier.

Reply:
👍